File: HERO C#/RC Mecanum Bot/Platform/Schedulers.cs
using CTRE.Phoenix.Tasking;

namespace HERO_Mecanum_Drive_Example.Platform
{
    public static class Schedulers
    {
        /* the schedulers.  Minimally you will likely want one periodic scheduler to run the normal tasks.
         * Additional schedulers could be ConsecutiveSchedulers for entire autonomous movements or preconfigured manuevers.
         * Use 'public static' because these are single objects. */

        public static ConcurrentScheduler PeriodicTasks = new ConcurrentScheduler(10);
    }
}

File: Java General/CANifier Demo/src/main/java/frc/robot/Platform/Schedulers.java
/** 
 * Simple Class containing schedulers used throughout project
 */
package frc.robot.Platform;

import com.ctre.phoenix.schedulers.ConcurrentScheduler;

public class Schedulers {
	/**
	 * The schedulers. Minimally you will likely want one periodic scheduler to
	 * run the normal tasks. Additional schedulers could be
	 * ConsecutiveSchedulers for entire autonomous movements or pre-configured
	 * maneuvers. Use 'public static' because these are single objects.
	 */
	public static ConcurrentScheduler PeriodicTasks = new ConcurrentScheduler();
}
